### ISSUE TITLE
Cannot `Clone Git Project as DataCurator ... get error: "a MessageNotUnderstood occurred (error 2010), a RwGenericUrl does not understand  #'fileName'"

### ISSUE THREAD

**Issue by dalehenrich** [dalehenrich/Rowan#211]
Loads okay as `SystemUser` (with some fixes to RowanSample1 configs on sample branch). 

Here's the stack:
```
MessageNotUnderstood >> defaultAction @2 line 3
MessageNotUnderstood (AbstractException) >> _signalWith: @5 line 25
MessageNotUnderstood (AbstractException) >> signal @2 line 47
RwGenericUrl (Object) >> doesNotUnderstand: @9 line 10
RwGenericUrl (Object) >> _doesNotUnderstand:args:envId:reason: @7 line 12
RwSpecification class >> fromUrl: @14 line 19
RwPrjCloneTool >> cloneSpecUrl:gitRootPath:useSsh: @2 line 4
RowanProjectService >> newGitProject:root:useSsh: @4 line 5
GsNMethod class >> _gsReturnToC @1 line 1
```

Suspicious that RwGenericUrl is doing a lookup (subclasses method uses **ClassOrganizer** that *does do lookups in the symbol list* ...

**Comment by dalehenrich** [comment]
Yep .... `RwUrl class >> urlClassForScheme:` was using `allSubclasses` and that involves **ClassOrganizer**

**Comment by dalehenrich** [comment]
... not out of the woods yet ... there *are* classes that need to be publicly available ... those that are read from **STON** files ... hadn't thought about that:) ... I've got two choices: make a number of classes used in STON public or arrange for the STON reader to peek in **RowanKernel** for missing classes ...

**Comment by dalehenrich** [comment]
final fix in b0aa42d ... lookup symbols in SystemUser's RowanKernel smbol dictionary ...
